### PR TITLE
AO-21: Removed duplicated icon keys from add-ons-to-index.json

### PR DESCRIPTION
**Pull request by anuar2k**
## Description of what I changed
I've removed duplicated `icon` keys in module list defined in `add-ons-to-index.json`. This was caused probably be merging two solutions for the same problem, probably https://issues.openmrs.org/browse/GCI-258

## Issue I worked on
To see the whole description of the problem, visit https://issues.openmrs.org/browse/AO-21

Link to the GCI task instance:
https://codein.withgoogle.com/dashboard/task-instances/6142415594323968/

### CODE REVIEW

**Comment by anuar2k** [comment]
@reubenvarghese1 Instead of removing other keys to match the pattern, I've decided to move them to keep the same icons and correct file structure at the same time. It's probably the best option of both worlds :P

File: src/main/resources/add-ons-to-index.json [hunk 1]
```diff
@@ -3191,7 +3173,6 @@
       "uid": "org.openmrs.module.smsmodule",
       "type": "OMOD",
       "name": "SMS Module",
-      "icon": "envelope-o",
```
Comment (reubenvarghese1) [inline src/main/resources/add-ons-to-index.json]: Guess you missed this one @anuar2k

File: src/main/resources/add-ons-to-index.json [hunk 2]
```diff
@@ -3191,7 +3173,6 @@
...
       "tags": [
         "messaging"
       ],
```
Comment (reubenvarghese1) [inline src/main/resources/add-ons-to-index.json]: And forgot to delete this one

File: src/main/resources/add-ons-to-index.json [hunk 3]
```diff
@@ -3454,7 +3434,6 @@
       "uid": "org.openmrs.module.usagestatistics",
       "type": "OMOD",
       "name": "Usage Statistics",
-      "icon": "bar-chart",
```
Comment (reubenvarghese1) [inline src/main/resources/add-ons-to-index.json]: And this one as well

**Pull request merged**